scanner/image/db/konflux.Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM registry.redhat.io/rhel8/postgresql-15:latest
+
+LABEL \
+    com.redhat.component="rhacs-scanner-v4-db-container" \
+    com.redhat.license_terms="https://www.redhat.com/agreements" \
+    description="Scanner v4 Database Image for Red Hat Advanced Cluster Security for Kubernetes" \
+    io.k8s.description="Scanner v4 Database Image for Red Hat Advanced Cluster Security for Kubernetes" \
+    io.k8s.display-name="scanner-v4-db" \
+    io.openshift.tags="rhacs,scanner-v4-db,stackrox" \
+    maintainer="Red Hat, Inc." \
+    name="rhacs-scanner-v4-db-rhel8" \
+    source-location="https://github.com/stackrox/stackrox" \
+    summary="Scanner v4 DB for RHACS" \
+    url="https://catalog.redhat.com/software/container-stacks/detail/60eefc88ee05ae7c5b8f041c" \
+    # We must set version label to prevent inheriting value set in the base stage.
+    # TODO(ROX-20236): configure injection of dynamic version value when it becomes possible.
+    version="0.0.1-todo"
+
+USER root
+
+COPY --chown=postgres:postgres \
+     scanner/image/db/scripts/docker-entrypoint.sh \
+     scanner/image/db/scripts/init-entrypoint.sh \
+     /usr/local/bin/
+
+RUN dnf upgrade -y --nobest && \
+    dnf install -y zstd && \
```

Comment:
Downstream image does not install `zstd` although I don't know if it's valid...
```suggestion
```

Reply:
I thought about this when removing the db-init bundle.  This is only required to handle decompressing that bundle.  My thinking was to leave it in since it may be easy to forget if we want to add back the init bundle.  The counterpoint is that it would only be one dev/test cycle to realize the package needs to be installed when adding back the init bundle, and in the mean time we have one more unused package in our image.

Reply:
I don't feel strongly about this but would suggest removing since having fewer orphan things requires less thinking cycles to explain months or years later.

Reply:
I went ahead and removed it to remain consistent with downstream: 3db3526dfdce71fa9eec3086793c10bde0920bc3